lite/examples/image_segmentation/android/app/src/main/AndroidManifest.xml

```diff
@@ -18,7 +18,6 @@
     xmlns:tools="http://schemas.android.com/tools"
     package="org.tensorflow.lite.examples.imagesegmentation">
 
-  <uses-sdk />
```

Comment:
Don't remove this line. It's needed for our internal build system.

Reply:
Done, thanks.